data/ideas.json

```diff
@@ -28,5 +28,11 @@
         "description": "Code up a clone of the windows calculator. It’s great practice for GUIs and basic math operation. And if it helps you solve your algebra homework - all the better!",
         "level": "basic",
         "tags": "calculator"
+    },
+    {
+        "title": "Feature maps and Filter Visualizer",
+        "description": "This repository is about the code to extract cnn layers and visualise the filters and featuremaps",
```

Comment:
What do you mean about `cnn`?

Reply:
convolutional neural network

Reply:
could you change `cnn` to `convolutional neural network`?

Reply:
Consider it done

Reply:
Did you commit and push it?

Reply:
Not yet